config/initializers/host.rb

```diff
@@ -1,8 +1,13 @@
-if ENV.key?("HOST")
-  Rails.application.config.hosts << ENV["HOST"]
+def set_up_host(host)
+  return unless host
```

Comment:
Should this `raise`? Seems bad.

Reply:
when you run `bin/dev`, and it runs at localhost:3000, that's when HOST isn't set and we need to return. when we deploy, if a request comes in and we didn't add the right domain to the `config.hosts` allow-list, everything will by a 500 and we will definitely notice. we could `raise if Rails.env.production?` though!